### DISCUSSION TITLE
Jupyter connectivity issue

### DISCUSSION THREAD

**Discussion by johnf329** [cooperrc/computational-mechanics#144] [Q&A]
Jupyter hub seems to be acting up.  It had been crashing frequently and right now I am not able to access my work. Is anybody else experiencing this issue?

**Comment by seancari1001** [comment]
I am having issues connecting as well. Its on and off.

**Reply by johnf329** [reply]
Yeah it crashed again for me. If I can connect again I am going to download my homework file and work on it locally with jupyter notebooks.
